### PR TITLE
Implemented user tax data extraction from emails

### PR DESCRIPTION
**Pull request by nocheacosador**
Implemented user tax data extraction from email using IMAP protocol.

### CODE REVIEW

File: Energy Saver/Properties/serviceDependencies.json [hunk 1]
```diff
@@ -7,6 +7,11 @@
     "mssql2": {
       "type": "mssql",
       "connectionId": "ConnectionStrings:EnergySaverTaxesContext"
+    },
+    "postgresql1": {
+      "type": "postgresql",
+      "connectionId": "DatabaseConnectionString",
+      "dynamicId": null
```
Comment (Leminode) [inline Energy Saver/Properties/serviceDependencies.json]: why'd you change these files?

**Review by Leminode — APPROVED** [review]
Don't know what's up with the serviceDependencies file but the rest looks good 👍

**Comment by Tauras-Rim** [comment]
The input page design could be better in my opinion, but overall good job

**Review by Tauras-Rim — APPROVED** [review]

**Review by priolette — APPROVED** [review]

File: Energy Saver/Properties/serviceDependencies.json (hunk 1 above)
Comment (nocheacosador) [reply Energy Saver/Properties/serviceDependencies.json]: Accidental commit

Comment (nocheacosador) [reply Energy Saver/Properties/serviceDependencies.json]: Removed it

**Pull request merged**